examples/src/java/org/partiql/examples/EvaluationJavaExample.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package org.partiql.examples;
+
+import com.amazon.ion.IonSystem;
+import com.amazon.ion.system.IonSystemBuilder;
+import kotlin.Unit;
+import org.jetbrains.annotations.NotNull;
+import org.partiql.examples.util.Example;
+import org.partiql.lang.CompilerPipeline;
+import org.partiql.lang.eval.Bindings;
+import org.partiql.lang.eval.EvaluationSession;
+import org.partiql.lang.eval.ExprValue;
+import org.partiql.lang.eval.Expression;
+import org.partiql.lang.syntax.SqlParser;
+
+import java.io.PrintStream;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonMap;
+
+public class EvaluationJavaExample extends Example {
```

Comment:
For this and others that apply: would be great to have a JavaDoc explaining the example.

Reply:
Ack